### ISSUE TITLE
Transaction list with the -i flag

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#38]
I've tested `transaction list fb93cce7aceb9f7fda228bc0c0c2eca8c56c09c1d846a04bd6a59cae2a895974 -c -i` and got the following output:

```
  File "~/.local/bin/pacli", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "~/.local/lib/python3.11/site-packages/pacli/__main__.py", line 441, in main
    fire.Fire({
  File "~/.local/lib/python3.11/site-packages/fire/core.py", line 141, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.11/site-packages/fire/core.py", line 475, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
                                ^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.11/site-packages/fire/core.py", line 691, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
                ^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.11/site-packages/pacli/extended_classes.py", line 1074, in list
    return ei.run_command(self.__list, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.11/site-packages/pacli/extended_interface.py", line 31, in run_command
    result = c(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.11/site-packages/pacli/extended_classes.py", line 1170, in __list
    pprint([t["txid"] for t in txes])
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.11/site-packages/pacli/extended_classes.py", line 1170, in <listcomp>
    pprint([t["txid"] for t in txes])
            ~^^^^^^^^
KeyError: 'txid'
```

The `transaction list mtMGkB2mVxzPNZCWHruRmG1SmsSJBtyCUD -i` as well as many others I've tested are regular.

The `transaction list -w -i` outputs the list of 1000 lines and then adds:
```
  # ...and 311 more elements
```
(I have 1311 transactions in my wallet)
If someone needs the full list what should he do?

**Comment by d5000** [comment]
The first bug is fixed with commit 6e01a7d.

The second problem is related to the "pretty printing" library, because the list is so long it gets truncated to 1000 elements (it can always be overridden with `-q` but then the list is uglier).

I've found the relevant parameter in the documentation, but setting it to None (to disable truncation) seems also to disable the "pretty printing" and outputs the same "ugly" list than with `-q`.

I've set the maximum number now to 100000, this seems to work (commit fac5871).

This problem may perhaps appear in other commands too, although I think `transaction list` outputs the longest lists we have.

I think also that longer lists than 100000 would never be really seen on a screen, so prettyprinting isn't really important in this case. In this case `-q` should be used. I left a little warning in the help string.
